components/_patterns/00-base/global/base/_04-base.scss

```diff
@@ -0,0 +1,5 @@
+main {
```

Comment:
Should use `.main` instead of hanging this on the element. Could cause issues when `main` is used elsewhere.